package.json

```diff
@@ -4,8 +4,10 @@
   "description": "Scheduling and Appointment Management for VITA",
   "main": "index.js",
   "scripts": {
-    "dev": "babel . -d dist --ignore node_modules/,dist/",
-    "production": "babel . -d dist --ignore node_modules/,dist/"
+    "transpile": "babel . -d dist --ignore node_modules/,dist/",
+    "cachebust": "hashmark dist/**/*.js {dir}/{name}-{hash}{ext} | sed 's/\\\\\\\\\\\\\\\\/\\\//g'  | replaceinfiles -s **/*.php -d {dir}/{name}{ext}",
```

Comment:
I'm not certain why it required so many frigging escaped backslashes... But it seems that it actually ended up replacing escaped characters twice. Once when one runs the original command (`npm run production`), and a second time when the command was actually ran by npm... so I had to double escape everything I wanted. Not pretty, but it works.